### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by travistrle**
This is a feature that helps website to authenticate with Google

### CODE REVIEW

**Comment by tracytrle** [comment]
Hey @travistrle , you need to add the python code as below
```python
for index in xrange(0, 5):
    print index
```

File: travistrle.txt [hunk 1]
```diff
@@ -8,5 +8,9 @@ Test closed #1 by Travis
 Start Branch feature1
 Travis is working on feature 1
 Added code by Tracy to feature1
+
+for index in xrange(0, 5):
```
Comment (travistrle) [inline travistrle.txt]: You shouldn't hard code with 5. You need to use built-in function len to check the len of list

Comment (tracytrle) [inline travistrle.txt]: You shouldn't hard code with 5. You need to use built-in function len to check the len of list

Comment (travistrle) [inline travistrle.txt]: Got it. Will change it now

**Comment by tracytrle** [comment]
So far so good, I will added my code and this issue will be close if it can pass the CI/CD

**Pull request merged**